### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by croqueGrec09**
includes commits for:
- ERMS-5276, addresses list has been reorganised
- ERMS-5359: display of discovery systems corrected
- ERMS-5413: query fix
- ERMS-5414: obsolete export menu entry removed

### CODE REVIEW

File: grails-app/views/organisation/_discoverySystemModal.gsp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </g:if>
+    <g:elseif test="${config == 'discoverySystemIndex'}">
+        <a class="ui button" data-ui="modal" href="#index">
+            <g:message code="org.discoverySystems.index.add"/>
+        </a>
+
+        <ui:modal id="index" message="org.discoverySystems.index.add">
+            <g:form class="ui form" url="[controller: 'organisation', action: 'addDiscoverySystem', id:org.id]" method="post">
+                <div class="field">
+                    <label><g:message code="org.discoverySystems.index.label"/>:</label>
```
Comment (klodav) [inline grails-app/views/organisation/_discoverySystemModal.gsp:32]: missing label **for** attribute

File: grails-app/views/organisation/_discoverySystemModal.gsp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<g:if test="${editable}">
+    <g:if test="${config == 'discoverySystemFrontend'}">
+        <a class="ui button" data-ui="modal" href="#frontend">
+            <g:message code="org.discoverySystems.frontend.add"/>
+        </a>
+
+        <ui:modal id="frontend" message="org.discoverySystems.frontend.add">
+            <g:form class="ui form" url="[controller: 'organisation', action: 'addDiscoverySystem',id:org.id]" method="post">
+                <div class="field">
+                    <label><g:message code="org.discoverySystems.frontend.label"/>:</label>
```
Comment (klodav) [inline grails-app/views/organisation/_discoverySystemModal.gsp:11]: missing label **for** attribute

**Review by klodav — CHANGES_REQUESTED** [review]
!

File: grails-app/views/organisation/_discoverySystemModal.gsp (hunk 1 above)
Comment (croqueGrec09) [reply grails-app/views/organisation/_discoverySystemModal.gsp]: fixed in [c2c585c]

File: grails-app/views/organisation/_discoverySystemModal.gsp (hunk 2 above)
Comment (croqueGrec09) [reply grails-app/views/organisation/_discoverySystemModal.gsp]: fixed in c2c585c

**Review by klodav — APPROVED** [review]

**Pull request merged**